### ISSUE TITLE
[Discussion] Auto weekly report for dubbo

### ISSUE THREAD

**Issue by Frankzhaopku** [apache/incubator-dubbo#3153]
I am a developer from Alibaba and I build a GitHub collaborate [robot](https://github.com/AlibabaDR/Collabobot) which is a framework to collaborate with GitHub projects.

Currently, this robot has a component to report weekly status of a repo as a issue, like [this](#3152). This weekly report will summarize the basic data of the repo and overview of merged pull requests, pr reviews and new contributors which is very helpful to get the general picture of Dubbo.

This robot has basic services like data provider, event manager, GitHub request client, GitHub hook, web service and scheduler. But with weekly report function only, we don't need to have any access authority of the repo and the robot will only play as a community developer who knows the repo very well.

WDYT.

**Comment by nobodyiam** [comment]
Looks great!

Is it possible to use this [robot](https://github.com/AlibabaDR/Collabobot) in other github projects?

**Comment by Frankzhaopku** [comment]
> Looks great!
> 
> Is it possible to use this [robot](https://github.com/AlibabaDR/Collabobot) in other github projects?

Of course, this robot is a collaborate framework which provides some basic services and you can use it on any project, I am still working on finish the basic functions and README file to help with quick start.

**Comment by LookforAnswer** [comment]
> I am a developer from Alibaba and I build a GitHub collaborate [robot](https://github.com/AlibabaDR/Collabobot) which is a framework to collaborate with GitHub projects.
> 
> Currently, this robot has a component to report weekly status of a repo as a issue, like [this](https://github.com/apache/incubator-dubbo/issues/3152). This weekly report will summarize the basic data of the repo and overview of merged pull requests, pr reviews and new contributors which is very helpful to get the general picture of Dubbo.
> 
> This robot has basic services like data provider, event manager, GitHub request client, GitHub hook, web service and scheduler. But with weekly report function only, we don't need to have any access authority of the repo and the robot will only play as a community developer who knows the repo very well.
> 
> WDYT.



> Already deploy robot for Dubbo weekly report.

周报机器人罢工了，需要调整

**Comment by Frankzhaopku** [comment]
> 周报机器人罢工了，需要调整@Frankzhaopku

对不起，由于我已经离开阿里了，目前这个机器人的部署已不再我的权力范围内了，我会尽快和社区沟通，看是否需要重新为 Dubbo 部署一个机器人
